ymlfmt/Dockerfile

```
@@ -1,9 +1,9 @@
-FROM python:3.7-alpine
+FROM python:3.8-alpine
```

I verified that nothing in this bump affects the usage of this image in https://github.com/docker-library/docs, so seems sane, although we should take the opportunity to pin as elsewhere:

```suggestion
FROM python:3.8-alpine3.10
```